src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar 100 bucks:
https://www.ebay.com/b/10Gbps-Internal-Network-Cards/20318/bn_742047

I guess that you have it already ?
http://www.clubic.com/reseau-informatique/wi-fi/actualite-819136-netgear-nighthawk-x10-wigig-plex-media-server-sfp.html
https://www.netgear.com/landings/ad7200/
https://support.apple.com/fr-fr/HT204154
That's leave with Apple or LG monitor:
https://en.wikipedia.org/wiki/List_of_Thunderbolt-compatible_devices#cite_note-50
Some LG and few other phones or Phablet.
And some TV like Panasonic [tc-58ax800u](http://shop.panasonic.com/support-only/TC-65AX800U.html) or tc-65ax800u using the [DisplayPort](https://en.wikipedia.org/wiki/DisplayPort)...